src/tests/userTickets/cancelUserTicket.test.ts

```diff
@@ -0,0 +1,196 @@
+import { it, describe, afterEach, assert } from "vitest";
+import {
+  executeGraphqlOperationAsUser,
+  insertCommunity,
+  insertEvent,
+  insertEventToCommunity,
+  insertTicket,
+  insertTicketTemplate,
+  insertUser,
+  insertUserToCommunity,
+  insertUserToEvent,
+} from "~/tests/__fixtures";
+import { clearDatabase } from "~/tests/__fixtures/databaseHelper";
+import {
+  CancelUserTicket,
+  CancelUserTicketMutation,
+  CancelUserTicketMutationVariables,
+} from "./cancelUserTicket.generated";
+
+afterEach(() => {
+  clearDatabase();
+});
+
+describe("User", () => {
+  it("Should cancel a user ticket", async () => {
```

Comment:
`NITPICK:`

Le pondría "Should cancel a user ticket if user is the owner of the ticket"